utilities/operations/sites/prefetchWobjs.js

```diff
@@ -0,0 +1,36 @@
+const searchHelper = require('utilities/helpers/searchHelper');
+const { App, PrefetchModel } = require('models');
+
+exports.showAllPrefs = async (data) => {
+  const { result, error } = await PrefetchModel.find({ type: data.type });
```

Comment:
1)what will happen if you have a million records?
2) does any of your indexes will work?